guidelines/enhancement_template.md

```diff
@@ -12,7 +12,8 @@ api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggreg
   - TBD
 creation-date: yyyy-mm-dd
 last-updated: yyyy-mm-dd
-status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced|informational
+jira link: # link to the Jira Feature or Epic ticket that corresponds to this enhancement
```

Comment:
Let's call this something more generic, since teams outside of Red Hat may use other tools for tracking. Maybe "tracking_link"?

Would it make sense to move this out of the "header" and into the body of the document somewhere?

Reply:
tracking link wfm

Reply:
let me play around with placement locally for a bit to see.